manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/MatchTest.scala

```diff
@@ -19,347 +19,305 @@
-      assertions = (p) => assertEquals(2, p.size)
-    )
-  }
+      p("The following graph is used for the examples below:")
+      graphViz()
+    }
+    section("Basic node finding") {
+      section("Get all nodes") {
+        p("By just specifying a pattern with a single node and no labels, all nodes in the graph will be returned.")
+        query("MATCH (n) RETURN n", assertAllNodesReturned) {
+          p("Returns all the nodes in the database.")
+          resultTable()
+        }
+      }
+      section("Get all nodes with a label") {
+        p("Getting all nodes with a label on them is done with a single node pattern where the node has a label on it.")
+        query("MATCH (movie:Movie) RETURN movie.title", assertAllMoviesAreReturned) {
+          p("Returns all the movies in the database.")
+          resultTable()
+        }
+      }
+      section("Related nodes") {
+        p("The symbol `--` means _related to,_ without regard to type or direction of the relationship.")
+        query("MATCH (director { name:'Oliver Stone' })--(movie) RETURN movie.title", assertWallStreetIsReturned) {
+          p("Returns all the movies directed by Oliver Stone.")
+          resultTable()
+        }
+      }
+      section("Match with labels") {
+        p("To constrain your pattern with labels on nodes, you add it to your pattern nodes, using the label syntax.")
+        query("MATCH (:Person { name:'Oliver Stone' })--(movie:Movie) RETURN movie.title", assertWallStreetIsReturned) {
+          p("Return any nodes connected with the +Person+ Oliver that are labeled +Movie+.")
+          resultTable()
+        }
+      }
+    }
+    section("Relationship basics") {
+      section("Outgoing relationships") {
+        p("When the direction of a relationship is interesting, it is shown by using `-->` or `<--`, like this:")
+        query("MATCH (:Person { name:'Oliver Stone' })-->(movie) RETURN movie.title", assertWallStreetIsReturned) {
+          p("Return any nodes connected with the +Person+ Oliver that are labeled +Movie+.")
+          resultTable()
+        }
+      }
+      section("Directed relationships and identifier") {
+        p("If an identifier is needed, either for filtering on properties of the relationship, or to return the relationship, this is how you introduce the identifier.")
+        query("MATCH (:Person { name:'Oliver Stone' })-[r]->(movie) RETURN type(r)", assertRelationshipIsDirected) {
+          p("Returns all outgoing relationships from Oliver.")
+          resultTable()
+        }
+      }
+      section("Match by relationship type") {
+        p("When you know the relationship type you want to match on, you can specify it by using a colon together with the relationship type.")
+        query("MATCH (wallstreet:Movie { title:'Wall Street' })<-[:ACTED_IN]-(actor) RETURN actor.name", assertAllActorsOfWallStreetAreFound) {
+          p("Returns all outgoing relationships from Oliver.")
```

Comment:
This is clearly wrong. It should say something like `Returns all actors that acted in the movie Wall Street.`